.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,55 @@
+name: Scorecard supply-chain security
```

Comment:
can we show some kind of badge in the readme for this scorecard?

Reply:
Done ;)